configure.ac

```diff
@@ -1970,6 +1970,8 @@ AS_IF([test x"$with_afl" = "xyes"],
   [afl=true],
   [afl=false])
 
+internal_cppflags="-DCAML_INTERNALS -I\$(ROOTDIR)/runtime $internal_cppflags"
```

Comment:
I know we do this elsewhere, but I'm a little bit "meh" about having embedded make variables being generated inside `configure` (it's occasionally awkward). Can I propose a slightly different approach here - `$internal_cppflags` is used exactly once in `$oc_cppflags` which is then used exactly once in `Makefile.config.in` to initialise `$(OC_CPPFLAGS)`. Could we just put absolute flags like this straight in `Makefile.config.in`, therefore: `OC_CPPFLAGS = -DCAML_INTERNALS -I$(ROOTDIR)/runtime @oc_cppflags@`?